pyflow/scene/from_ipynb_conversion.py

```diff
@@ -60,7 +60,12 @@ def get_blocks_data(
         else:
             block_type: str = cell["cell_type"]
 
-            text: str = cell["source"]
+            text: List[str] = []
+
+            if type(cell["source"]) == list:
```

Comment:
```suggestion
            if isinstance(cell["source"], list):
```
Use isinstance preferably